components/Article/PayNote.js

```diff
@@ -88,37 +88,59 @@ export const MAX_PAYNOTE_SEED = Math.max(
 
 const goTo = route => Router.pushRoute(route).then(() => window.scrollTo(0, 0))
 
-const getTryVariation = (seed, { query }) => {
-  const variations =
-    TRY_VARIATIONS_CAMPAIGN[query.campaign || query.utm_campaign] ||
-    TRY_VARIATIONS
-  const variation = getElementFromSeed(variations, seed)
+const filterNotes = (payNotes, filterFn) => {
+  if (!payNotes || !payNotes.length) return
+  const filtered = payNotes.filter(filterFn)
+  return filtered.length && filtered
+}
+
+const getTryNotes = payNotes => filterNotes(payNotes, note => note.isTrynote)
+
+const getBuyNotes = payNotes => filterNotes(payNotes, note => !note.isTrynote)
+
+const getCustomNote = (seed, noteType, customNotes) => {
   return {
-    keyShort: variation,
-    key: `article/tryNote/${variation}`,
-    cta: 'try'
+    keyShort: `custom-${noteType}note`,
+    payNote: getElementFromSeed(customNotes, seed, MAX_PAYNOTE_SEED),
```

Comment:
```suggestion
    custom: getElementFromSeed(customNotes, seed, MAX_PAYNOTE_SEED),
```
Otherwise this leads to `payNote.payNote` which is highly confusing.